src/index.js

```diff
@@ -58,28 +75,60 @@ class App {
       return;
     }
 
+    const name = place.name;
+
     const location = {
       lat: place.geometry.location.lat(),
       lon: place.geometry.location.lng()
     }
 
-    this.createForecast(location);
+    this.buildPageByQuery(location);
+  }
+
+  handleClearSearch() {
+    this.elements.input.value = '';
+    this.elements.input.focus();
   }
 
-  createForecast(location) {
-    axios.get(API_URL, { params: { ...location, key: API_KEY } })
+  buildPageByQuery(params) {
+    axios.get(API_URL, { params: { ...params, key: API_KEY } })
       .then(response => {
-        this.forecast.createForecast(response.data.data, response.data.city_name);
-        console.log(response.data);
+        const data = {
+          forecast: response.data.data,
+
+          lat: response.data.lat,
+          lon: response.data.lon
+
+        }
+
+        this.forecast.createForecast(data.forecast);
+
+        this.getCityName(data.lat, data.lon)
```

Comment:
You need to use `getCityName` by location only when you get this location from navigator, but you don't need to do this when you get place data from google autocomplete, because city name already exists in that response